src/bors/handlers/pr_events.rs

```diff
@@ -14,11 +14,11 @@ pub(super) async fn handle_pull_request_edited(
 ) -> anyhow::Result<()> {
     let pr = &payload.pull_request;
     let pr_number = pr.number;
-    db.update_pr_base_branch(repo_state.repository(), pr_number, &pr.base.name)
-        .await?;
 
     // If the base branch has changed, unapprove the PR
     let Some(_) = payload.from_base_sha else {
+        db.update_pr_base_branch(repo_state.repository(), pr_number, &pr.base.name)
```

Comment:
It's the opposite :sweat_smile: The PR has changed when `from_base_sha` is `Some`, not `None`.